docs/IPFS.md

```diff
@@ -33,7 +33,7 @@ When you provide no additional details to cURL then cURL will:
 If cURL fails you'll be presented with an error message and a link to this page to the option most applicable to solving the issue.
 
 ### --ipfs-gateway argument
-You can also provide a `--ipfs-gateway` argument to cURL. This overrules any other gateway setting. cURL won't fallback to the other options if the provided gateway didn't work.
+You can also provide a `--ipfs-gateway` argument to cURL. This overrules any other gateway setting. CURL won't fallback to the other options if the provided gateway didn't work.
```

Comment:
We call it "curl".

Reply:
That fails your checks. A line must start with a capital letter. I'm fine calling it Curl, curl, cURL or even a mixture thereof. As long as it doesn't fail compilation.

Reply:
No, because `curl` and `libcurl` are accepted as-is after periods in markdown files.

Reply:
If not, the test is buggy.

Reply:
Ohh sweet! Will change then :)